test/run-tests

```diff
@@ -110,13 +110,12 @@ def handle_transient_httperrors(error):
         except Exception:
             github_msg = ""
         else:
-            print("W: GitHub: " + github_msg)
+            logging.error("GitHub: %s", github_msg)
```

Comment:
Should this be a warning?  I'm assuming that's what the `W:` meant in the original message